zod.go

```diff
@@ -777,6 +777,8 @@ func (c *Converter) validateString(validate string) string {
 			switch part {
 			case "required":
 				validateStr.WriteString(".min(1)")
+			case "omitempty":
+				validateStr.WriteString(".optional()")
```

Comment:
We should ignore this i.e. 
use the case but `validateStr.WriteString(".optional()")` is not required

We are already generating nullable which is what we need to generate, also we may need it for numbers and slices as  well